### DISCUSSION TITLE
__fzf_search_current_dir does not find any files in the current directory

### DISCUSSION THREAD

**Discussion by juliohm1978** [PatrickF1/fzf.fish#100] [Q&A] [answered]
For some reason, `__fzf_search_current_dir` always shows 0 matches in any current directory.

![image](https://user-images.githubusercontent.com/4145456/104104501-58deef80-5287-11eb-9933-f0db6d4d3f72.png)

Any ideas on what's causing this?

**Comment by PatrickF1** [comment]
Hi Julio, can you try executing it using Ctrl+F?

**Comment by juliohm1978** [comment]
Yes. Using the shortcut gives me the same result. I had to change the shortcut from the default (https://github.com/PatrickF1/fzf.fish/discussions/99), so I'm pretty sure `__fzf_search_current_dir` is what's been called here.

**Comment by juliohm1978** [comment]
Strangely, I can use `fzf` natively and it does list all files in the current directory.

![image](https://user-images.githubusercontent.com/4145456/104109601-69569080-52ae-11eb-8320-ddf17f5ed9e5.png)

**Comment by PatrickF1** [comment]
Just to be sure, did you already go through these troubleshooting steps? https://github.com/PatrickF1/fzf.fish#file-search-feature-does-not-work

**Comment by juliohm1978** [comment, marked as answer]
omg, bingo!

[`fdclone`](https://www.linuxlinks.com/fdclone-console-based-lightweight-file-manager/) was installed on my system, conflicting the expected `/usr/bin/fd` from `fdfind`.

I just removed fdclone and created a symlink `/usr/bin/fd` that points to `/usr/bin/fdfind`.

Works like a charm now. Thank you!

**Comment by PatrickF1** [comment]
Wow who knew there was another fd! Thanks for reporting the tip. I'll add this to the troubleshooting guide.
